### DISCUSSION TITLE
flink cdc 是否支持postgresql ddl 捕获

### DISCUSSION THREAD

**Discussion by beninzhao** [apache/flink-cdc#1397] [Q&A]
最近用debezium做pg数据库的实时数据同步，因为表结构经常变化，更改比较麻烦，请问一下flink cdc 是否可以实现表结构变化后的实时动态同步呢？大佬求解答啊

**Comment by taylorThunderbolt** [comment]
不支持。
具体请看：https://debezium.io/documentation/reference/1.5/connectors/postgresql.html#postgresql-overview
